spec/model/event_spec.rb

```diff
@@ -35,4 +35,85 @@
       expect(event.errors[:online]).to_not be_empty
     end
   end
+
+  context 'Event has a chat_server' do
+    it 'Event has no chat_server' do
+      event = Event.new
+      event.chat_server = nil
+      event.chat_server_url = nil
+
+      event.save
+      expect(event.valid_chat_server).to be true
+      expect(event.errors[:chat_server]).to be_empty
+    end
+    it 'Event has a chat_server and a chat_server_url errors should be empty' do
+      event = Event.new
+      event.chat_server = 'Discord'
+      event.chat_server_url = 'https://www.google.com'
+
+      event.save
+      expect(event.valid_chat_server).to be true
+      expect(event.errors[:chat_server]).to be_empty
+    end
+
+    it 'If event has a chat_server_url but no chat_server error should exist' do
+      event = Event.new
+      event.chat_server = nil
+      event.chat_server_url = 'https://www.google.com'
+
+      event.save
+      expect(event.valid_chat_server).to be false
+      expect(event.errors[:chat_server]).to_not be_empty
+    end
```

Comment:
@blomquistr I actually prefer this rule to be implemented. While we are suppressing the display if `chat_server` is nil (reminds me, should change that to blank), this prevents someone from getting confused why their chat_server_url isn't displaying when they didn't give a chat server.